data-analysis.js

```diff
@@ -0,0 +1,76 @@
+const url = 'https://dummyjson.com/users';
+getData(url)
```

Comment:
I would like to have one empty line after const

Reply:
an empty string is added